packages/web/src/pages/sign-up-page/pages/SignUpPage.tsx

```diff
@@ -1,20 +1,43 @@
-import { useCallback } from 'react'
+import { useCallback, useEffect } from 'react'
 
-import { Button, ButtonType, Text } from '@audius/harmony'
-import { Form, Formik } from 'formik'
+import { useEmailInUse } from '@audius/common/src/api/signUp'
+import {
+  Button,
+  ButtonType,
+  Divider,
+  Flex,
+  IconArrowRight,
+  Text,
+  TextInput
+} from '@audius/harmony'
+import { useFormik } from 'formik'
+import { useDispatch } from 'react-redux'
+import { z } from 'zod'
+import { toFormikValidationSchema } from 'zod-formik-adapter'
 
-import { TextField } from 'components/form-fields'
+import audiusLogoColored from 'assets/img/audiusLogoColored.png'
 import { Link } from 'components/link'
-import { SIGN_IN_PAGE } from 'utils/route'
+
+import { setValueField } from '../../../common/store/pages/signon/actions'
+import PreloadImage from '../../../components/preload-image/PreloadImage'
```

Comment:
can do same for components, hooks, and utils

Reply:
Fixed 👍